<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EvoVisForm</class>
 <widget class="QWidget" name="EvoVisForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>620</width>
    <height>307</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Crossing Minimization</string>
  </property>
  <widget class="QGraphicsView" name="graphicsView_graphdrawing">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>381</width>
     <height>281</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonExportGraphVisualization">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>161</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Export graph visualization</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_24">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Crossing number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_26">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Variance</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_variance">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_crossing">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>431</width>
     <height>21</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>0</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_loading">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>120</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Processing, please wait...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
